test/extended/cli/debug.go

```diff
@@ -0,0 +1,38 @@
+package cli
+
+import (
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[cli] oc debug", func() {
+	oc := exutil.NewCLI("oc-debug", exutil.KubeConfigPath())
+	templatePath := exutil.FixturePath("testdata", "test-cli-debug.yaml")
+
+	g.JustBeforeEach(func() {
+		g.By("waiting for builder service account")
+		err := exutil.WaitForBuilderAccount(oc.KubeClient().Core().ServiceAccounts(oc.Namespace()))
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		g.By("calling oc create -f " + templatePath)
+		err = oc.Run("create").Args("-f", templatePath).Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		exutil.WaitForAnImageStreamTag(oc, oc.Namespace(), "busybox", "latest")
+		o.Expect(err).NotTo(o.HaveOccurred())
+	})
+
+	g.It("should print the container entrypoint/command", func() {
```

Comment:
@smarterclayton @bparees I don't see how this can neatly be a test/cmd test as it stands because oc debug waits for the debug pod to start.  The options I see are: change to an integration test on getContainerImageCommand(), drop the test, leave it as it is.  Do you see another alternative?  How do you want me to proceed?